c_src/eleveldb.cc

```diff
@@ -73,13 +78,17 @@ static ErlNifFunc nif_funcs[] =
     {"async_iterator", 3, eleveldb::async_iterator},
     {"async_iterator", 4, eleveldb::async_iterator},
 
-    {"async_iterator_move", 3, eleveldb::async_iterator_move}
+    {"async_iterator_move", 3, eleveldb::async_iterator_move},
+
+    {"property_cache", 2, eleveldb::property_cache},
+    {"set_metadata_pid", 1, eleveldb::set_metadata_pid}
```

Comment:
Same as my comment in erleveldb.erl, I'd like this to be a more generalized mechanism.